### PR TITLE
update inveniordm v12

### PR DESCRIPTION
**Pull request by fenekku**
- **make compatible with InvenioRDM v12.0**
- **:package: release v0.5.0**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -42,7 +42,7 @@ readme = "README.md"
 requires-python = ">=3.8"
 scripts = {galter-subjects-utils = "galter_subjects_utils.cli:main"}
 urls = {Repository = "https://github.com/galterlibrary/galter-subjects-utils"}
-version = "0.4.2"
+version = "0.5.0"
```
Comment (Meowcenary) [inline pyproject.toml:45]: 🥳

File: README.md [hunk 2]
```diff
@@ -26,6 +26,7 @@ This repository follows [semantic versioning](https://semver.org/) indexed on in
 | --------------------- | ----------------------- |
 | 0.3.X                 | 11.X                    |
 | 0.4.X                 | 11.X                    |
+| 0.5.X                 | 12.X                    |
```
Comment (Meowcenary) [inline README.md:29]: ‼️

**Review by Meowcenary — APPROVED** [review]
Another straightforward one. Thanks! 👍

**Review by sharpattack — APPROVED** [review]
Looks good, thanks.

**Pull request merged**